src/prefect/cli/flow_run.py

```diff
@@ -139,13 +142,36 @@ async def cancel(id: UUID):
 
 
 @flow_run_app.command()
-async def logs(id: UUID):
+async def logs(
+    id: UUID,
+    head: bool = typer.Option(
+        False,
+        "--head",
+        "-h",
+        help=f"Show the first {LOGS_WITH_LIMIT_FLAG_DEFAULT_NUM_LOGS} lines of logs instead of all logs.",
+    ),
+    num_lines: int = typer.Option(
```

Comment:
I think this should be `num_logs`

Reply:
Good catch. I fixed it an additional few occurrences of 'lines'